src/vivarium/framework/values.py

```diff
@@ -471,7 +479,9 @@ def get_value(self, name: str) -> Pipeline:
             (frequently just a :class:`pandas.Index` representing the
             simulants).
         """
-        return self._pipelines[name]  # May create a pipeline.
+        pipeline = self._pipelines.get(name) or Pipeline(name)
```

Comment:
I understand the expected behavior but why isn't this returning a bool?

Reply:
https://realpython.com/python-or-operator/#non-boolean-contexts

Reply:
Oh cool didn't know you could do that with or